apps/web/src/lang/fr.json

```diff
@@ -6786,10 +6786,6 @@
     "defaultMessage": "Rôles des membres",
     "description": "Label for the input to add roles to a user's team membership"
   },
-  "eW7I5E": {
-    "defaultMessage": "Sélectionner les rôles...",
-    "description": "Placeholder text for role selection input"
-  },
```

Comment:
@petertgiles 
translation added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5904/commits/268e08b374ad8589dee53eac5e916ee95ac5c5a6
this deletion appears to have been a duplicate that was removed by one of the npm intl commands, translation check is passing

Reply:
I ran a few intl things to try so that's why I am not sure which one removed it 😅

Reply:
:+1: